src/box/memtx_tree.cc

```diff
@@ -930,6 +965,120 @@ memtx_tree_index_get_internal(struct index *base, const char *key,
 	return 0;
 }
 
+/**
+ * Implementation of @tree_iterator_position for general and multikey indexes.
```

Comment:
Nit: Please don't use `@` or other Doxygen-like notation in internal comments. They are useless and only make the comments look inconsistent, because we don't usually use them.

Reply:
Removed all doxygen-style comments from the patch.